src/Platform/Microsoft.Testing.Platform/Builder/TestApplicationBuilder.cs

```diff
@@ -68,6 +68,9 @@ internal TestApplicationBuilder(
 
     internal ITelemetryManager Telemetry => _testHostBuilder.Telemetry;
 
+    [Obsolete("Remove in v2. Avoid breaking change with the rename of the property. See https://github.com/microsoft/testfx/issues/5015")]
```

Comment:
Do we want to maybe go strong and make it error?

```suggestion
    [Obsolete("Remove in v2. Avoid breaking change with the rename of the property. See https://github.com/microsoft/testfx/issues/5015", error: true)]
```